pyads/ads.py

```diff
@@ -178,10 +179,13 @@ def set_local_address(ams_netid: Union[str, SAmsNetId]) -> None:
 def add_route(adr: Union[str, AmsAddr], ip_address: str) -> None:
     """Establish a new route in the AMS Router (linux Only).
 
-    :param adr: AMS Address of routing endpoint as str or AmsAddr object
+    :param adr: AMS Address of routing endpoint as str or AmsAddr object. If an
```

Comment:
I propose we use None to indicate that no ams address is provided. So the data type should be Optional[Union[str, AmsAddr]]

Reply:
Sounds good, unfortunately we cannot give it a default value because of the argument ordering (so as to maintain backwards compatibility), but we can at least change the type annotation.